### PR TITLE
1102/filter fills trades

### LINKED ISSUE #1102: Filter /trades page (not trade that's done)
Part of #1032 

Filter `/trades` page

### PR DESCRIPTION
**Pull request by W3stside**
Closes #1102 

Also filters by "market" for Fills

To test:
1. head to `/trades` or into the `Fills` tab @ root
2. search for a market

**You'll have to delete application cache as the `trades` are cached and this current iteration depends on the hook rebuilding the `trades` list to include `market` prop**

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr1116--dexreact.review.gnosisdev.com

File: src/api/exchange/ExchangeApi.ts [hunk 1]
```diff
@@ -105,7 +105,16 @@ export interface ExchangeApi extends DepositApi {
   cancelOrders(params: CancelOrdersParams): Promise<Receipt>
 }
 
-export interface DetailedAuctionElement extends AuctionElement {
+// Where market = 'WETH-TUSD'
+export interface Market {
```
Comment (anxolin) [inline src/api/exchange/ExchangeApi.ts:109]: Do we need to introduce artificial derived data just for the filter?
Can't we just define this logic in the filter function itself?

This way, also the filtering by market can have self contained in the function the logics like: 
- accept: WETH-TUSD
- accept: WETHTUSD
- accept: WETH/TUSD

**Comment by W3stside** [comment]
> Does not really work for trades:
> screenshot_2020-06-17_10-23-27
> 
> And results are named `orders` o.O

you need to delete application cache as it's using the cache trades list. there won't be `market` populated there. 

and that makes what @anxolin requested more appealing

File: src/api/exchange/ExchangeApi.ts (hunk 1 above)
Comment (W3stside) [reply src/api/exchange/ExchangeApi.ts:109]: @anxolin fixed in #1119 via commits [129c9c](https://github.com/gnosis/dex-react/pull/1119/commits/129c9c12b2da48410c2d4c4d3bc79b9b294f603a) and [
ab85b52](https://github.com/gnosis/dex-react/pull/1119/commits/ab85b52a7acaee7c53007f16350668567d657d58)

**Comment by W3stside** [comment]
@anxolin @alfetopito made requested changes in #1119

**Pull request merged**